### DISCUSSION TITLE
Mitigate double counting of LCA approach

### DISCUSSION THREAD

**Discussion by leon-hard** [electricitymaps/electricitymaps-contrib#6589] [General]
Hello together,

Out of my point of view, the LCA approach as used by EM is subject to an inherent double counting in electricity based emissions considered in the LCAs which are used for its emission factors.

**Simple example: PV module is produced in China and installed in China**

Production caused let's keep it simple 1 tCO2 by usage of electricity for the energy intensive processes. This 1 tCO2 is already accounted for by the emissions for the power as calculated for the year 2024 for China.The lifetime of the PV module is now assumed to be 30 years, the LCA will distribute this emissions over its lifetime and the PV module will be accounted with 1 t / 30 emissions at each year. After its lifetime, the reported emissions will be 2 t CO2, however, in reality, only 1 tCO2 was emitted, i.e. double counting appeared.

**Possible solution** LCA factors should not take into account electricity based emissions

What are your ideas regarding this problem?

Thank you very much!

Best regards,
Leonhard

**Comment by madsnedergaard** [comment]
Hey @leon-hard, my answer somehow got lost here (I most likely forgot to press "Comment") - sorry about that!

The LCA approach is not without it's flaws, but to my knowledge there is no scientific sources available to support making the split. Are you aware of any?

It is the same situation with nuclear where ore enrichment is also done using electricity (as far as I am aware).
